From 82b75921413502374ad23d7a9d93b9e9021c0742 Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Thu, 28 Apr 2016 15:58:52 -0700
Subject: [PATCH 1/2] build: Specify C language explicitly for CMake

If the build system doesn't have a C++ compiler installed, CMake will fail
with a line like:
Check for working CXX compiler: CMAKE_CXX_COMPILER_FULLPATH-NOTFOUND

As flatcc is explicitly C-only, let's be sure we can use it without a C++
compiler installed.

Signed-off-by: Steve deRosier <steve.derosier@lairdtech.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6157f96..b9f5bc3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@
 #cmake_minimum_required (VERSION 2.8.11)
 cmake_minimum_required (VERSION 2.8)
 
-project (FlatCC)
+project (FlatCC C)
 
 #
 # NOTE: when changing build options, clean the build using:
-- 
1.9.1

